core/src/components/title/title.tsx

```diff
@@ -22,9 +22,8 @@ export class ToolbarTitle implements ComponentInterface {
 
   /**
    * The size of the toolbar title.
-   * Only applies in `ios` mode.
    */
-  @Prop() size?: 'large' | undefined;
+  @Prop() size?: 'large' | 'small';
```

Comment:
should this be

```suggestion
  @Prop() size?: 'large' | 'small' | undefined;
```

instead?